README.md

```diff
@@ -15,51 +9,99 @@ Nextcloud Talk lowers the barrier for communication and lets your team connect a
 
-For more information please review the [guidelines for contributing](https://github.com/nextcloud/talk-android/blob/master/CONTRIBUTING.md) to this repository.
+Please review the [guidelines for contributing](/CONTRIBUTING.md) to this repository.
+
+More information how to contribute: [https://nextcloud.com/contribute/](https://nextcloud.com/contribute/)
+
 
-### Testing
+## Start contributing :hammer_and_wrench:
+
+Make sure you read [SETUP.md](/SETUP.md) and [CONTRIBUTING.md](/CONTRIBUTING.md) before you start working on this project. But basically: fork this repository and contribute back using pull requests to the master branch.
+Easy starting points are also reviewing [pull requests](https://github.com/nextcloud/talk-android/pulls) and working on [starter issues](https://github.com/nextcloud/talk-android/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+
+### Testing :test_tube:
 
 So you would like to contribute by testing? Awesome, we appreciate that very much. 
 
 To report a bug for the alpha or beta version, just create an issue on github like you would for the stable version and
  provide the version number. Please remember that Google Services are necessary to receive push notifications. 
  
-#### Beta versions (Release Candidates)
+ 
+#### Beta versions (Release Candidates) :package:
+
 
 ##### via Google Play
+
 Sign up at [Google Play Beta channel](https://play.google.com/apps/testing/com.nextcloud.talk2) to get Release
- Candidates via Google Play.
+ Can
+ didates via Google Play.
+
 
 ##### via github
+
 You can also get the Release Candidates at [github releases](https://github.com/nextcloud/talk-android/releases).
 
+
 #### Alpha versions
 
+
 ##### via Google Play
+
 To become an alpha tester you have to be signed up for the [Google Play Beta channel](https://play.google.com/apps/testing/com.nextcloud.talk2) 
 and additionally you have to join the [Alpha testing Google Group](https://groups.google.com/g/nextcloud-android-talk-alpha-testing). 
 After that you will receive the alpha versions via the Play Store (initially, this might take some minutes after
  signing up). However, in the Play Store the app will still be named "Nextcloud Talk (Beta)" even if you are an alpha tester, but you will receive the alpha versions.
 If a beta was released that is newer than the alpha version, you will get the beta in the alpha channel.
  
+ 
 ##### via Download page
+
 In addition to google play, the alpha and beta apps can also be obtained from the Nextcloud [Download page](https://download.nextcloud.com/android/talk-alpha/)
 Please make sure to remember that these versions might contain bugs and you don't use them in production.
 
-### Apply a license
+
+### Apply a license :scroll:
```

Comment:
Can we move this and "Signt your work" to the contibuting.md?
Really lot of unnecessary text on the front page

Reply:
DONE